include/solver_enums.h

```diff
@@ -63,6 +63,8 @@ enum SolverAttribute
   THEORY_DATATYPE,
   // supports quantifiers
   QUANTIFIERS,
+  // supports getting the operator of a term
+  GET_OP,
```

Comment:
who doesn't support this?

Reply:
Good point, it used to be CVC4 but that has changed. And @lstuntz just added support for it in the Z3 backend so currently all solvers support it. Addressed with 2e5a182f6851b357b71f8bd9380efc077cfae34e.